src/components/accession2/Accession2View.tsx

```diff
@@ -99,6 +99,47 @@ export default function Accession2View(props: Accession2ViewProps): JSX.Element
     }
   };
 
+  const getAbsoluteQuantity = () => {
+    if (accession && accession.remainingQuantity) {
+      if (accession.remainingQuantity.units === 'Seeds') {
+        return (
+          <Typography display='flex' sx={{ marginRight: '5px' }}>
+            <Typography sx={{ fontWeight: 600, marginRight: '5px' }}>{accession.remainingQuantity.quantity}</Typography>
+            <Typography>{strings.CT}</Typography>
```

Comment:
we aren't using 'COUNT'?

Reply:
In all designs shows "ct", but I can change it if you think it should be count

Reply:
That's fine, we can ask Ming later and update if needed.